tor_core/initialize.py

```diff
@@ -78,6 +79,18 @@ def configure_logging(config, log_name='transcribersofreddit.log'):
     else:
         logging.info('Not running with Bugsnag!')
 
+    if config.sentry_api_url:
+        # All we have to do is create the client object in order to start
+        # sending things to Sentry. We're saving the client object here though
+        # so that we can use it for specific things in the future if we ever
+        # want to.
+        config.sentry = Client(config.sentry_api_url)
+
+    if config.sentry_api_url:
+        logging.info('Sentry enabled!')
```

Comment:
I feel like the line above for configuring the client can be combined with this section handling the log messages. Is there a reason you wanted to keep them separate?

Reply:
mmmmnot really. They can be combined.

Reply:
Then I'll go ahead and fix up this stuff and merge?

Reply:
Yep, sounds good to me!